### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by rianp**
## Changes made:

1. Add spin animation, color change, and sound effect when the user clicks on the cell
2. Add background image animation 
3. Make the game status output Bold so it is easier to read
4. Add Banner image to Title 
5. Add styling to instructions so it's easier to read

## Not included:

1. Background music loop (game soundtrack)
2. Randomized sound effects for cell click
3. Sound effects for board size selection buttons

## Comments
Incorporating audio elements and sound effects into the game significantly enhances its enjoyment. But, there is a noticeable delay in the background audio loop, prompting me to defer its implementation. This decision stems from recognizing that a more significant underlying issue might be at play, potentially warranting its own separate task or ticket. The same consideration for time and ticket sizing applies to integrating sound effects for button selections within the menu. Moreover, exploring the option of introducing randomization to the cell click sound effects to prevent monotony is worth considering.

I'd also like to know whether my approach of employing a CSS-based side-scrolling technique for the background image is optimal for background animation. While my online research indicates that this method is an efficient way to achieve this effect, I am curious if others might have valuable input.

[ticket](https://trello.com/c/F6M8eR8X)

Screenshot 2023-08-08 at 12 09 30 PM

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -67,6 +86,11 @@
     inset -2px -2px 4px rgba(0, 0, 0, 0.1);
 }
 
+.game-status-text {
+  color: black;
+  font-weight: bold;
+}
+
```
Comment (ranizilpelwar) [inline src/App.css:93]: FE changes are greatly helped by including a video recording in the PR. This way users don't need to run the branch locally to know how the UI changes, esp styling, appear. You don't need to do this now but its something that looks really great on a client project.

File: src/board/Cell.css [hunk 2]
```diff
@@ -2,15 +2,24 @@
...
     inset 2px 2px 4px rgba(255, 255, 255, 0.3),
     inset -2px -2px 4px rgba(0, 0, 0, 0.1);
+  transition: transform 0.9s;
+}
+
+.cell.flipped {
+  transform: rotateY(360deg);
+  color: black;
+  background-color: rgb(255, 255, 255);
 }
```
Comment (ranizilpelwar) [inline src/board/Cell.css:25]: Wow you really got into the complex css stuff; pretty cool. You have just out-learned css compared to me!

File: src/board/Cell.jsx [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
 import "./Cell.css";
 import PropTypes from "prop-types";
+import { isValidMove } from "../validation/validateMove";
+
+function Cell({ cellSpot, handleClick, gameStatus }) {
+  console.log(cellSpot);
+  const [isFlipped, setIsFlipped] = useState(false);
+
+  const handleCellClick = () => {
+    const audio = new Audio(process.env.PUBLIC_URL + "/cell.wav");
```
Comment (ranizilpelwar) [inline src/board/Cell.jsx]: One enhancement is that you can keep the OI stuff outside of this component. So having a function for audio that does the work on the edge of your app and then calling the function in your component makes sense from a decoupling standpoint.
One ex of a refactor:
if (isValidMove(cellSpot) & (gameStatus === "Ongoing")) {
      setIsFlipped(!isFlipped);
      playCellSelectedSound();
    }


So, playCellSelectedSound() is defined outside of your component in another ts file.

**Review by ranizilpelwar — APPROVED** [review]
Glad you had fun with this one!

File: src/App.css (hunk 1 above)
Comment (rianp) [reply src/App.css:93]: oh good idea!

File: src/board/Cell.jsx (hunk 3 above)
Comment (rianp) [reply src/board/Cell.jsx]: oh true! thank you!

File: src/board/Cell.jsx [hunk 4]
```diff
@@ -1,9 +1,26 @@
+import React, { useState } from "react";
 import "./Cell.css";
 import PropTypes from "prop-types";
+import { isValidMove } from "../validation/validateMove";
+
+function Cell({ cellSpot, handleClick, gameStatus }) {
+  console.log(cellSpot);
```
Comment (clairenguyen) [inline src/board/Cell.jsx:8]: did you mean to leave this comment in? always good to review your changes before making any commits :)

File: src/game/isGameOver.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import { getBoardStatus } from "../board/getBoardStatus";
+
+export function isGameOver() {
+  return getBoardStatus() !== "Ongoing";
+}
```
Comment (clairenguyen) [inline src/game/isGameOver.js]: This method feels a bit funky to me. There should probably be a test around this. 

As a standalone function, how would isGameOver know what `getBoardStatus` should be? If you tested this, you might find that getBoardStatus actually expects an argument ` export function getBoardStatus(board) `

--

Actually, after looking at your code again, I don't think `isGameOver` is used anywhere? Should this file actually not be here?

**Review by clairenguyen — APPROVED** [review]
Overall, good job. Like Rani said, glad you had fun with this! 

My main bit of feedback is around proofing/checking your changes both before and after you make your PR.  After I push my changes to a remote branch, and right before I make my PR, I like to double check the diff on github (or bitbucket, or whatever platform) one more time. Sometimes seeing the changes in a view that's different from my IDE or terminal helps me notice some of the things I missed.

**Comment by clairenguyen** [comment]
Also, probably should have updated the "wip" commit, either by amending it or squashing it. 

image

**Review by fifikim — APPROVED** [review]
looks gorgeous! i'm in love!

File: src/board/Cell.jsx (hunk 4 above)
Comment (rianp) [reply src/board/Cell.jsx:8]: hmm. I'm not seeing a comment in the code. Maybe I'm missing something?

File: src/game/isGameOver.js (hunk 5 above)
Comment (rianp) [reply src/game/isGameOver.js]: Oh! You're right! That was some legacy code from the spike. Thanks for catching that. I'll be more conscientious of this in the future.

**Pull request merged**